base/pkg.jl

```diff
@@ -13,9 +13,17 @@ type PkgError <: Exception
     msg::AbstractString
 end
 
-for file in split("dir types reqs cache read query resolve write entry git")
-    include("pkg/$file.jl")
-end
+include("pkg/dir.jl" )
```

Comment:
No worries. For me personally, an editor with multiline support achieves DRY in this instance. Plus it's easier to parse this way.